packages/transcript-view/dev_tools/transcript-view-dev-options.ts

```diff
@@ -0,0 +1,316 @@
+      bubbles: true,
+      composed: true,
+    });
+    this.dispatchEvent(event);
+  }
+
+  private changeShowContextZones(e: Event): void {
+    const target = e.target as HTMLFormElement;
+    this.showContextZones = target.checked;
+    const event = new CustomEvent('showContextZonesChanged', {
+      detail: { showContextZones: this.showContextZones },
+      bubbles: true,
+      composed: true,
+    });
+    this.dispatchEvent(event);
+  }
+
+  private setNormalTimerDelay(): void {
+    this.changeTimerDelay(1000);
+  }
+
+  private setFastTimerDelay(): void {
+    this.changeTimerDelay(500);
+  }
+
+  private setFasterTimerDelay(): void {
+    this.changeTimerDelay(250);
+  }
+
+  private changeTimerDelay(delay: number): void {
+    this.timerDelay = delay;
+    this.startTimer();
+  }
+
+  private startStopTimer(): void {
+    if (this.timerRunning) {
+      this.stopTimer();
+    } else {
+      this.startTimer();
+    }
+  }
+
+  private startTimer(): void {
+    this.stopTimer();
+    this.timerRunning = true;
+    this.timer = window.setInterval(() => {
+      if (this.currentTime >= 445) {
+        this.stopTimer();
+        return;
+      }
+      this.currentTime += 1;
+      this.emitCurrentTimeChangedEvent();
+    }, this.timerDelay);
+  }
+
+  private stopTimer(): void {
+    this.timerRunning = false;
+    window.clearInterval(this.timer);
+  }
+
+  handleCurrentTimeSlide(e: Event): void {
+    this.stopTimer();
+    const target = e.target as HTMLFormElement;
+    const seconds = parseFloat(target.value);
+    this.currentTime = seconds;
+    this.emitCurrentTimeChangedEvent();
+  }
+
+  emitCurrentTimeChangedEvent(): void {
+    const event = new CustomEvent('currentTimeChanged', {
+      detail: { currentTime: this.currentTime },
+      bubbles: true,
+      composed: true,
+    });
+    this.dispatchEvent(event);
+  }
+
+  handleTopContextSlide(e: Event): void {
+    const target = e.target as HTMLFormElement;
+    const height = target.value;
+    this.topContextHeight = height;
+    const event = new CustomEvent('topContextHeightChanged', {
+      detail: { height },
+      bubbles: true,
+      composed: true,
+    });
+    this.dispatchEvent(event);
+  }
+
+  handleBottomContextSlide(e: Event): void {
+    const target = e.target as HTMLFormElement;
+    const height = target.value;
+    this.bottomContextHeight = height;
+    const event = new CustomEvent('bottomContextHeightChanged', {
+      detail: { height },
+      bubbles: true,
+      composed: true,
+    });
+    this.dispatchEvent(event);
+  }
+
+  handleTranscriptSizeSlide(e: Event): void {
+    const target = e.target as HTMLFormElement;
+    const height = target.value;
+    this.transcriptHeight = height;
+    const event = new CustomEvent('transcriptHeightChanged', {
+      detail: { height },
+      bubbles: true,
+      composed: true,
+    });
+    this.dispatchEvent(event);
+  }
+
+  showHideBottomContext(e: Event): void {
```

Comment:
**Suggestion**: similar DRY request as above, with the methods `showHideBottomContext` and `showHidetopContext` and the `CustomEvent` name param.